api/controllers/AttendanceController.ts

```diff
@@ -31,6 +35,18 @@ export class AttendanceController {
     return { error: null, attendances };
   }
 
+  @Get('/user/:uuid')
+  async getAttendancesForUser(@Params() params: UuidParam,
+    @AuthenticatedUser() currentUser: UserModel): Promise<GetAttendancesForEventResponse> {
+    if (params.uuid === currentUser.uuid) {
+      return this.getAttendancesForCurrentUser(currentUser);
+    }
+    const user = await this.userAccountService.findByUuid(params.uuid);
+    if (!user.canSeeAttendance) throw new ForbiddenError();
+    const attendances = await this.attendanceService.getAttendancesForUser(user);
```

Comment:
nit: let's move lines 44-45 to the attendance service call, so that all business logic is under that function call, and so that database calls are under that same transaction that takes place in `getAttendancesForUser`.